hardware/Schematic-Design-Guide.md

```diff
@@ -96,41 +125,59 @@ The matching effort strives to simultaneously achieve several goals:
-  ![](files/HW-Schematic-Design-Guide/Sub-G-Match-LT-500MHz.png)
-  ![](files/HW-Schematic-Design-Guide/Sub-G-Match-MT-500MHz.png)
+   <div align="center">
+    <img src="files\HW-Schematic-Design-Guide\Sub-G-Match-Topology.png">  
+   </div>
+  
+  The matching circuits are classified into 2 types due to the balun difference. One is for low frequency band(<500MHz), and the other is for high frequency band(>500MHz)
+  <div align="center">
+   <img src="files\HW-Schematic-Design-Guide\Sub-G-Match-LT-500MHz.png">  
+  </div>
+
+
+  <div align="center">
+  <img src="files\HW-Schematic-Design-Guide\Sub-G-Match-MT-500MHz.png">  
+  </div>
 
   And more the matching circuits are different from components values which classified with output power levels and sub frequency bands:
 
   Matching components value table:
-  ![](files/HW-Schematic-Design-Guide/Sub-G-Match-Circuits.png)
+  <div align="center">
+   <img src="files\HW-Schematic-Design-Guide\Sub-G-Match-Circuits.png">  
+  </div>
+
   Performance table:
-  ![](files/HW-Schematic-Design-Guide/Sub-G-Tested-Performance.png)
+  <div align="center">
+   <img src="files\HW-Schematic-Design-Guide\Sub-G-Tested-Performance.png">  
+  </div>
 
 - Matching circuits design for 2.4GHz band
   AN930 describes the 2.4GHz matching method in detail.The 2.4 GHz front end has a unified, single-ended TX and RX pin (2G4RF_IOP), so the TX and RX paths are tied together internally. The 2G4RF_ION TX pin has to be grounded at the pin. Externally, a single-ended matching network and harmonic filtering are required.
 
   2 main 2.4 GHz matching topologies are presented here:
 
   A ladder 2-element LC match for up to 10 dBm power levels:
-
-  ![](files/HW-Schematic-Design-Guide/2.4G-2element.png)
-  
+  <div align="center">
+   <img src="files\HW-Schematic-Design-Guide\2.4G-2element.png">  
+  </div>
+ 
   A ladder 4-element LCLC match for up to 20 dBm power levels:
-
-  ![](files/HW-Schematic-Design-Guide/2.4G-4element.png)
-
+  <div align="center">
+   <img src="files\HW-Schematic-Design-Guide\2.4G-4element.png">  
+  </div>
+  
 The performances:
    Tx output power and harmonics:
-
-  ![](files/HW-Schematic-Design-Guide/2.4G-Tx-TestResults.png) 
+  <div align="center">
+   <img src="files\HW-Schematic-Design-Guide\2.4G-Tx-TestResults.png">  
+  </div>
   
    Rx sensitivity:
-   ![](files/HW-Schematic-Design-Guide/2.4G-Rx-TestResults.png) 
-  
+   <div align="center">
+    <img src="files\HW-Schematic-Design-Guide\2.4G-Rx-TestResults.png">  
+   </div>
 
-## Customize the schematic design to fit the application
+# 5. Customize the schematic design to fit the application
    There are several points should be considered before a project's schematic design:
 - Which protocol do you want to use?
   Proprietary can provide most flexible communication protocol. Zigbee and BLE are more professional with network communication and control. Z-Wave is more safety for door lock application.
```

Comment:
Does it make sense to add a final section mentioning Silicon Labs Apps teams helping with design reviews for customers?

Reply:
This is basic tech. training material for customer. For design review support, I think it can be described in a separate KBA or sales support material.